pkg/kubelet/dockertools/docker.go

```diff
@@ -306,8 +306,10 @@ func ConnectToDockerOrDie(dockerEndpoint string) DockerInterface {
 
 func milliCPUToShares(milliCPU int64) int64 {
 	if milliCPU == 0 {
-		// zero milliCPU means unset. Use kernel default.
-		return 0
+		// Docker converts zero milliCPU to unset, which maps to kernel default
+		// for unset: 1024. Return 2 here to really match kernel default for
+		// zero milliCPU.
+		return 2
```

Comment:
I was just concerned that users may not know that the default behavior is, and they'd be baffled when their pod is not getting enough cpu time. If we can make it clear in the api/docs, that'd be better of course.